src/Synapse/Synapse/Common/HelpMessages.cs

```diff
@@ -265,10 +265,16 @@ public static class HelpMessages
 
         public const string RoleDefinitionId = "Id of the Role that is assigned to the principal.";
 
+        public const string RoleAssignmentItemType = "Item type of the role assignment:'bigDataPools', 'integrationRuntimes', 'linkedServices', 'credentials'.";
```

Comment:
>'bigDataPools', 'integrationRuntimes', 'linkedServices', 'credentials'. [](start = 87, length = 71)

IIRC, the possible value set can be displayed automatically. Could you please help confirm by `Get-Help New-AzSynapseWorkspace`?

Reply:
![image](https://user-images.githubusercontent.com/46309533/112271189-93b4c900-8cb5-11eb-898f-1c7f3a1fe1a3.png)
it only shows this